lib/osf-components/addon/components/file-browser/delete-file-modal/component.ts

```diff
@@ -42,15 +51,16 @@ export default class DeleteFileModal extends Component<Args> {
         try {
             await taskFor(item.delete).perform();
             this.failedFiles.removeObject(item);
-            this.deletedFiles.push(item);
+            this.deletedFiles.pushObject(item);
         } catch (e) {
             captureException(e);
         }
     }
 
     @action
     reset() {
-        this.deleteItemTasks = [];
+        this.files = [];
```

Comment:
I'm a little but confused as to why `this.files` is needed. It seems like it's an alias for `@items`, but is getting emptied out whenever the modal is closed? What happens when a user opens the delete modal, cancels/closes the modal before confirming the delete?